recipe/meta.yaml

```diff
@@ -5,30 +5,36 @@ package:
   version: {{ version }}
 
 source:
-  fn: yasm-{{ version }}.tar.gz                                                    # [unix]
-  url: http://www.tortall.net/projects/yasm/releases/yasm-{{ version }}.tar.gz     # [unix]
-  sha1: b7574e9f0826bedef975d64d3825f75fbaeef55e                                   # [unix]
-  fn: yasm.exe                                                                     # [win]
-  url: http://www.tortall.net/projects/yasm/releases/yasm-{{ version }}-win32.exe  # [win32]
-  sha1: 8374a6a1d6a240a3baa771ffd61b3bc096ccd11f                                   # [win32]
-  url: http://www.tortall.net/projects/yasm/releases/yasm-{{ version }}-win64.exe  # [win64]
-  sha1: f7872c6db3b4d93fd5a51717364d438108473149                                   # [win64]
+  fn: yasm-{{ version }}.tar.gz
+  url: http://www.tortall.net/projects/yasm/releases/yasm-{{ version }}.tar.gz
+  sha1: b7574e9f0826bedef975d64d3825f75fbaeef55e
 
 build:
   number: 0
-  skip: true  # [unix and py3k]
+  skip: true  # [(unix and py3k) or (win and py36)]
```

Comment:
Python 3.5 and 3.6 use the same VC runtime. So I just picked one and skipped the other.